### PR TITLE
Official 1.15 Release Docs 

### PR DESCRIPTION
**Pull request by makoscafee**
## 🛑 DO NOT MERGE 🛑 UNTIL KUBERNETES 1.15 IS RELEASED
Target Date: 2019-06-17

## Release Deadline

Date | Description
-- | --
MAY 30 | Open placeholder PRs
JUNE 04 | PRs ready for review
JUNE 10 | All PRs reviewed and ready to merge

## More information
contact: @MAKOSCAFEE @simplytunde @daminisatya @christianh814

### CODE REVIEW

File: content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_etcd-healthcheck-client.md [hunk 1]
```diff
@@ -1,98 +1,35 @@
 
-Generates the client certificate for liveness probes to healtcheck etcd
+Renew the certificate for liveness probes to healtcheck etcd
```
Comment (sftim) [inline content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_etcd-healthcheck-client.md]: Spotted a typo here. Best to fix it after release?

Comment (makoscafee) [reply content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_etcd-healthcheck-client.md]: I think so based on the time we have before release... I would suggest this be fixed after release.. @jimangel thoughts?

Comment (sftim) [reply content/en/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_renew_etcd-healthcheck-client.md]: I logged https://github.com/kubernetes/website/issues/14985

**Comment by kbhawkey** [comment]
Is it too late to update the reference pages with 1.15? Also, I think there is a generated page that is missing from the left nav.

**Comment by makoscafee** [comment]
> Is it too late to update the version and reference pages with 1.15? Also, I think there is a generated page that is missing from the left nav.

@kbhawkey You mean updating config.toml right?

**Comment by kbhawkey** [comment]
> 
> 
> > Is it too late to update the version and reference pages with 1.15? Also, I think there is a generated page that is missing from the left nav.
> 
> @kbhawkey You mean updating `config.toml` right?

Yes, as well as a few pages in the reference section.

**Comment by makoscafee** [comment]
@kbhawkey  is it possible/ok to do page generation after this has landed to master? that after merging this branch to master?

**Pull request merged**